### PR TITLE
Update meetings-with-SIP-H323-devices.md

### PR DESCRIPTION
**Pull request by NancyForero**
Corrected some typos, removed unnecessary parameters clarification and reworded the options

### CODE REVIEW

**Comment by NancyForero** [comment]
I suggest under Requriements this to be the #2 final text:


2. You must have a suitable agreement or license plan with one of the certified [Cloud Video Interop providers](../cloud-video-interop.md) (CVI) that offers SIP and H.323 dialing, such as Pexip or Poly. Please contact your chosen CVI provider for more information.

File: Teams/rooms/meetings-with-SIP-H323-devices.md [hunk 1]
```diff
@@ -23,22 +23,20 @@ description: Learn about how to set up Teams Rooms to receive and place calls to
...
 
 ## Requirements
 
 To enable your Teams Rooms on Windows to use SIP and H.323 dialing:
 
 1. The Teams Room account that is used to sign in to the device must have a Teams Room Pro license assigned.
-2. You must have an signed agreement with one of the certified [Cloud Video Interop providers](../cloud-video-interop.md) (CVI). However, you have two options:
-- **Option 1:** Sign up for a calling plan from a CVI provider that supports SIP and H.323 dialing.
-- **Option 2:** Buy your own licenses from a CVI provider that supports SIP and H.323 dialing. 
+2. You must have a signed agreement with one of the certified [Cloud Video Interop providers](../cloud-video-interop.md) (CVI) that offers this feature like for example, Poly and Pexip. Please reach out to your chosen CVI provider for more information.
```
Comment (JohanFreelancer9) [inline Teams/rooms/meetings-with-SIP-H323-devices.md:39]:
```suggestion
2. You must have a signed agreement with one of the certified [Cloud Video Interop (CVI) providers](../cloud-video-interop.md) that offers this feature like, for example, Poly and Pexip. Reach out to your chosen CVI provider for more information.
```

File: Teams/rooms/meetings-with-SIP-H323-devices.md [hunk 2]
```diff
@@ -59,11 +57,11 @@ New-CsTeamsRoomVideoTeleConferencingPolicy -Identity "TurnOnSIPH323" -Enabled $t
 ```
 >[!Important]
 >
-> The `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx` entry in the command is a placeholder for a GUID that is provided by the CVI vendor.
+> The `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx` entry in the command is a placeholder for a GUID that is provided by the CVI provider that offers this feature.
 
 >[!Note]
 >
-> `ReceiveExternalCalls`, `ReceiveInternalCalls`, `PlaceExternalCalls` and `laceInternalCalls` parameters are disabled by default. They take only two values either `Enabled` or `Disabled`. If it is set to `Enabled`, then the `AreaCode` parameter is required. This value is provided by your CVI partner.
+> `ReceiveExternalCalls` and `ReceiveInternalCalls` parameters are disabled by default. They take only two values either `Enabled` or `Disabled`. If it is set to `Enabled`, then the `AreaCode` parameter is required. This value is provided by the CVI provider that offers this feature.
```
Comment (JohanFreelancer9) [inline Teams/rooms/meetings-with-SIP-H323-devices.md:64]:
```suggestion
> The `ReceiveExternalCalls` and `ReceiveInternalCalls` parameters are disabled by default. They take only two values, either `Enabled` or `Disabled`. If it is set to `Enabled`, the `AreaCode` parameter is required. This value is provided by the CVI provider that offers this feature.
```

File: Teams/rooms/meetings-with-SIP-H323-devices.md [hunk 3]
```diff
@@ -79,7 +77,7 @@ Set-CsTeamsRoomVideoTeleConferencingPolicy -Identity `TurnOnSIPH323` -ReceiveExt
 >
 > Identity is required.
 
-To apply a policy to a Teams Rooms devie, you need to run the following command.
+To apply a policy to a Teams Rooms device, you need to run the following command.
```
Comment (JohanFreelancer9) [inline Teams/rooms/meetings-with-SIP-H323-devices.md:80]:
```suggestion
To apply a policy to a Teams Rooms device you need to run the following command.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NancyForero, kindly commit the few minor suggested changes. Many thanks.

**Pull request merged**